common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -139,21 +140,15 @@ def test_hidden_content_before_due(self):
         self.assertIn("'banner_text': None", html)
 
     @freeze_time(DAY_AFTER_TOMORROW)
-    @ddt.data(
-        (None, 'subsection'),
-        ('Homework', 'homework'),
-    )
-    @ddt.unpack
-    def test_hidden_content_past_due(self, format_type, expected_text):
+    def test_hidden_content_past_due(self):
         progress_url = 'http://test_progress_link'
-        self._set_sequence_format(self.sequence_4_1, format_type)
+        self._set_sequence_format(self.sequence_4_1, None)
```

Comment:
Since the message no longer uses the `format_type`, you should be able to delete the `_set_sequence_format` helper method altogether now.